packages/ingest/src/generate-lesson-plans/parseBatchLessonPlan.ts

```diff
@@ -37,6 +38,8 @@ export function parseBatchLessonPlan(line: unknown) {
     lessonPlan = CompletedLessonPlanSchema.parse(
       JSON.parse(maybeLessonPlanString),
     );
+
+    lessonPlan.keyStage = parseKeyStage(lessonPlan.keyStage);
```

Comment:
Nitpick: I personally have a preference for this style, but it's not important!

```suggestion
    const parsedLessonPlan = CompletedLessonPlanSchema.parse(
      JSON.parse(maybeLessonPlanString),
    );

    lessonPlan = {
      ...parsedLessonPlan,
      keyStage: parseKeyStage(lessonPlan.keyStage)
    };
```

Reply:
Also, shouldn't `let lessonPlan` have a type?

Reply:
I'm easy wrt to the style. I prefer not to mutate in general, but seeing has we're reassigning variables in the same block ...

Have changed though!

Reply:
Oh I see what you mean now ... I reassigned with the parse result. I'll update that 👍

Reply:
but

```ts
      keyStage: parseKeyStage(lessonPlan.keyStage)
```

(from your version) becomes

```ts
      keyStage: parseKeyStage(parsedLessonPlan.keyStage)
```